### ISSUE TITLE
Regular SAML TRs are treated as federation TRs by web UI

### ISSUE THREAD

**Issue by aliaksander-samuseu** [GluuFederation/oxTrust#1837]
Package: gluu-server-4.0-82.rc1.centos7 (Backend: OpendDJ)

When a single SP TR is created, it's "Relationship's Type" property in web UI will contain "site federated with THIS_TR_NAME" kind of string, what doesn't make much sense:
![Selection_030](https://user-images.githubusercontent.com/11802046/65287513-2e29ce80-db4c-11e9-832e-099c88468c4e.jpg)
Instead, it should be something like this:

1. For child TRs of some fed TR, it would  “SP federated with Name-Of-Parent-Fed-TR”
2. For fed TRs - "Federation"
3. For single SP TRs: "Standalone Service Provider" or just "Service Provider"

Also (not sure whether it's a separate issue, or direct result of web UI treating every TR as fed TR), you can't simply remove it, as an error will pop up:
![Selection_031](https://user-images.githubusercontent.com/11802046/65287516-34b84600-db4c-11e9-953d-750ab24e045c.jpg)
You can delete it only after deactivating it. Such thing should only happen when a user tries to delete a fed TR **which has existing child TRs based on it** - to prevent removing a lot of TRs due to a single mistake. All single SP TRs, **including child TRs of some fed TR**, should be removed without this additional step (there should still be a standard confirmation dialogue for this action, which works as it should as of now).

**Comment by syntrydy** [comment]
![image](https://user-images.githubusercontent.com/7513418/65333764-ff960d00-dbb8-11e9-8838-fe31f815bae2.png)
